www/consolelog.php

```diff
@@ -1,4 +1,5 @@
 <?php
+include 'common.inc';
```

Comment:
`require_once WWW_PATH . '/common.inc';`

Might be a little nicer here? Guarantees a single load, throws an error if there's no file found, and makes sure we can move it around with the new path! 😃

Reply:
the thing is autoload doesn't load `constants.inc` for the legacy code, so it will have to be __DIR__ when common.inc is included for now :(

Reply:
it's `require_once __DIR__ . '/common.inc';` now